apps/ci/ci-codestyle.sh

```diff
@@ -36,21 +36,21 @@ for check in ${!multiLineRegexChecks[@]}; do
     fi
 done
 
-declare -A classTtypeIdChecks=(
+declare -A getTypeIdChecks=(
     ["GetTypeId\(\)[[:space:]]*==[[:space:]]*TYPEID_PLAYER"]="Use IsPlayer() instead of GetTypeId() == TYPEID_PLAYER. Check the lines above"
     ["GetTypeId\(\)[[:space:]]*==[[:space:]]*TYPEID_CORPSE"]="Use IsCorpse() instead of GetTypeId() == TYPEID_CORPSE. Check the lines above"
     ["GetTypeId\(\)[[:space:]]*==[[:space:]]*TYPEID_ITEM"]="Use IsItem() instead of GetTypeId() == TYPEID_ITEM. Check the lines above"
     ["GetTypeId\(\)[[:space:]]*==[[:space:]]*TYPEID_DYNAMICOBJECT"]="Use IsDynObject() instead of GetTypeId() == TYPEID_DYNAMICOBJECT. Check the lines above"
 )
 
-EXCLUDE_PATTERN="--exclude=src/server/game/Entities/Object/Object.h"
+EXCLUDE_PATTERN="src/server/game/Entities/Object/Object.h"
 
-for check in ${!classTtypeIdChecks[@]}; do
+for check in ${!getTypeIdChecks[@]}; do
     echo "  Checking RegEx: '${check}'"
 
-    if grep -P -r -I -n ${check} ${EXCLUDE_PATTERN} src; then
+    if grep -P -r -I -n --exclude=${EXCLUDE_PATTERN} ${check} src; then
```

Comment:
${EXCLUDE_PATTERN} doesn't need to be wrapped in quotes?

Reply:
Probably. I dont write often Bash code. I will fixe it tonight.

Reply:
Arf, doesn't work. i don't understand why the cli souldn't exclude Object.h...